cmd/agent/android/README.md

```diff
@@ -0,0 +1,39 @@
+# Datadog Agent 6 on Android
+
+## Requirements
+
+-  android sdk and ndk (installed with android studio for example)
+- `gomobile` (through `go get golang.org/x/mobile/cmd/gomobile`) [godoc](https://godoc.org/golang.org/x/mobile/cmd/gomobile)
+- jdk (warning: on macos `brew cask install java8`, jdk10 provided by Oracle doesn't work)
+
+
+## Before building
+
+- `ANDROID_HOME` environment variable set to the android sdk path (if installed with android studio:  `$HOME/Android/Sdk` on linux and `$HOME/Library/Android/sdk` on macos).
+- initialize gomobile with `gomobile init -ndk /path/to/ndk` (ndk is at `$ANDROID_HOME/ndk-bundle` if installed with android studio).
```

Comment:
This is taken care of by the `inv deps` task if you add `--android`.  Should probably do it this way, because it also cleans up vendoring in `gomobile`

Reply:
Okay iI'll change this